kratos/processes/variational_distance_calculation_process.h

```diff
@@ -451,7 +451,7 @@ class VariationalDistanceCalculationProcess : public Process
         r_distance_model_part.Conditions().clear();
         r_distance_model_part.Elements().clear();
 
-        r_distance_model_part.SetProcessInfo( base_model_part.pGetProcessInfo() );
+        r_distance_model_part.SetProcessInfo(  base_model_part.pGetProcessInfo() );
```

Comment:
I think that you accidentally modified this. Would you mind to revert it?

Reply:
Done. Thank you for the hint.